model/customlist.py

```diff
@@ -215,6 +221,9 @@ def entries_for_work(self, work_or_edition):
             )
         return qu
 
+    def update_size(self):
+        self.size = len(self.entries)
```

Comment:
I'm not sure if this method is actually needed, but could be useful for repair purposes.